src/tileset_anim.c

```diff
@@ -237,62 +237,62 @@ static const void *const gUnknown_0837BB04[] =
 
 static const u8 *const gTilesetAnimTable_Mauville_0A[] =
 {
-    TilesetAnimTiles_Mauville_0_0,
-    TilesetAnimTiles_Mauville_0_0,
-    TilesetAnimTiles_Mauville_0_1,
-    TilesetAnimTiles_Mauville_0_2,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_3,
-    TilesetAnimTiles_Mauville_0_2,
-    TilesetAnimTiles_Mauville_0_1,
+    gTilesetAnims_Mauville0a_Frame0,
+    gTilesetAnims_Mauville0a_Frame0,
+    gTilesetAnims_Mauville0a_Frame1,
+    gTilesetAnims_Mauville0a_Frame2,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame3,
+    gTilesetAnims_Mauville0a_Frame2,
+    gTilesetAnims_Mauville0a_Frame1,
 };
 
 static const u8 *const gTilesetAnimTable_Mauville_1A[] =
```

Comment:
I"m noticing that the names of a few of these table aren't matching the names of the entries in the table. For example, this one has `Mauville_1A` in its name, but the entries have `Mauville0b`.  Are they supposed to match?